eidpdata/src/main/java/uk/ac/ucl/eidp/data/PoolStrategy.java

```diff
@@ -32,24 +21,22 @@
 import javax.naming.InitialContext;
 import javax.naming.NamingException;
 import javax.sql.DataSource;
-import uk.ac.ucl.eidp.data.jaxb.StatementGenerator;
-import uk.ac.ucl.eidp.data.jaxb.StatementGenerator.Parameter;
-import uk.ac.ucl.eidp.data.jaxb.StatementProducer;
+
 
 /**
  *
- * @author David Guzman <d.guzman at ucl.ac.uk>
+ * @author David Guzman {@literal d.guzman at ucl.ac.uk}
  */
 @Stateless
 @NodeQualifier(NodeType.POOL)
-public class PoolStrategy implements DBMappingStrategy {
+public class PoolStrategy implements DbMappingStrategy {
 
   private Properties properties;
-  private final String DS_JNDI_NAME = "datasource-jndi-name";
-  private final String RS_SCROLL_TYPE = "resultset-scroll-type";
-  private final String RS_CONCURRENCY_MODE = "resultset-concurrency-mode";
-  private final String SQL_DIALECT_PROP = "uk.ac.ucl.eidp.data.jaxb.JaxbSqlStatement";
-  private String SQL_DIALECT = "uk.ac.ucl.eidp.data.jaxb.JaxbSqlAnsi";
+  private final String dsJndiName = "datasource-jndi-name";
+  private final String rsScrollType = "resultset-scroll-type";
```

Comment:
\[FindBugs\] INFO: SS: Unread field: uk\.ac\.ucl\.eidp\.data\.PoolStrategy\.rsScrollType; should this field be static?